features/measurements/impl/src/test/java/org/opennms/netmgt/measurements/filters/impl/LoadDataTrendLineTest.java

```diff
@@ -0,0 +1,156 @@
+/*******************************************************************************
```

Comment:
need to update copyright headers here as well

Reply:
updated